JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -122,6 +112,32 @@ module Microsoft.ApplicationInsights.Context {
             }
         }
 
+        private initializeAutomaticSessionWithData(params) {
+            if (params.length > 0) {
+                this.automaticSession.id = params[0];
+            }
+
+            try {
+                if (params.length > 1) {
+                    var acq = +params[1];
+                    this.automaticSession.acquisitionDate = +new Date(acq);
+                    this.automaticSession.acquisitionDate = this.automaticSession.acquisitionDate > 0 ? this.automaticSession.acquisitionDate : 0;
+                }
+
+                if (params.length > 2) {
+                    var renewal = +params[2];
+                    this.automaticSession.renewalDate = +new Date(renewal);
+                    this.automaticSession.renewalDate = this.automaticSession.renewalDate > 0 ? this.automaticSession.renewalDate : 0;
+                }
+            } catch (e) {
+                _InternalLogging.throwInternalNonUserActionable(LoggingSeverity.WARNING, "Error parsing ai_session cookie, session will be reset: " + Util.dump(e));
```

Comment:
Considering that we now have logic to pull cookie out of local storage, we have just increased our points of failure. I would think we would want to track these failures. My suggestion would be to change this to CRITICAL. 
@MaxShehovtsov @olegsych thoughts?